core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-traffic-split-add/dot-experiments-configuration-traffic-split-add.component.ts

```diff
@@ -110,8 +117,13 @@ export class DotExperimentsConfigurationTrafficSplitAddComponent implements OnIn
         });
     }
 
-    get variants(): FormArray {
-        return this.form.get('variants') as FormArray;
+    /**
+     * Check if any of the variants has an error.
+     * @returns boolean
+     * @memberof DotExperimentsConfigurationTrafficSplitAddComponent
+     */
+    variantsHasErrors(): boolean {
+        return this.variants.controls.some((variant) => !!variant.errors);
```

Comment:
You can use direct `this.variants.status` it retrieve `VALID` | `INVALID` (Type FormControlStatus) direct to the FormArray